src/main/java/org/springframework/data/couchbase/core/ReactiveCouchbaseTemplate.java

```diff
@@ -45,26 +52,63 @@ public class ReactiveCouchbaseTemplate implements ReactiveCouchbaseOperations, A
 	private final PersistenceExceptionTranslator exceptionTranslator;
 	private final ReactiveCouchbaseTemplateSupport templateSupport;
 	private ThreadLocal<PseudoArgs<?>> threadLocalArgs = new ThreadLocal<>();
-	private QueryScanConsistency scanConsistency;
+	private final QueryScanConsistency scanConsistency;
 
 	public ReactiveCouchbaseTemplate(final CouchbaseClientFactory clientFactory, final CouchbaseConverter converter) {
-		this(clientFactory, converter, new JacksonTranslationService());
+		this(clientFactory, converter, new JacksonTranslationService(), null);
 	}
 
 	public ReactiveCouchbaseTemplate(final CouchbaseClientFactory clientFactory, final CouchbaseConverter converter,
-			final TranslationService translationService) {
+																	 final TranslationService translationService) {
 		this(clientFactory, converter, translationService, null);
 	}
 
 	public ReactiveCouchbaseTemplate(final CouchbaseClientFactory clientFactory, final CouchbaseConverter converter,
-			final TranslationService translationService, QueryScanConsistency scanConsistency) {
+			final TranslationService translationService, final QueryScanConsistency scanConsistency) {
 		this.clientFactory = clientFactory;
 		this.converter = converter;
 		this.exceptionTranslator = clientFactory.getExceptionTranslator();
 		this.templateSupport = new ReactiveCouchbaseTemplateSupport(this, converter, translationService);
 		this.scanConsistency = scanConsistency;
 	}
 
+	public <T> Mono<T> save(T entity, String... scopeAndCollection) {
+		Assert.notNull(entity, "Entity must not be null!");
```

Comment:
the contract of a mono is that it should only to work if subscribed. So I think this should all be wrapped into a Mono.defer(() -> ... )